pkg/gitlab/gitlab.go

```diff
@@ -0,0 +1,23 @@
+package gitlab
+
+import (
+	"errors"
+	"regexp"
+)
+
+// GetLatestCommit returns the commit matching the HEAD SHA of the parent merge request
```

Comment:
"parent" might be confusing. What I meant was the merge request that was the "parent" of the commit, but I'm thinking it might be superfluous to call it that. I think the relationship between them can probably be inferred pretty easily.